In [3]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

In [7]:
gdp_df = pd.read_csv('../data/gdp_percapita.csv')

In [8]:
gdp_df

Country or Area  Year        Value  Value Footnotes
0        Afghanistan  2020  1970.560169              NaN
1        Afghanistan  2019  2065.036235              NaN
2        Afghanistan  2018  2033.804389              NaN
3        Afghanistan  2017  2058.400221              NaN
4        Afghanistan  2016  2057.067978              NaN
...              ...   ...          ...              ...
7171        Zimbabwe  1994  2603.927369              NaN
7172        Zimbabwe  1993  2420.092791              NaN
7173        Zimbabwe  1992  2437.152908              NaN
7174        Zimbabwe  1991  2733.696626              NaN
7175        Zimbabwe  1990  2652.129209              NaN

[7176 rows x 4 columns]

In [9]:
type(gdp_df)

pandas.core.frame.DataFrame

Question 5. There are 7176 row and 4 columns in the gdp_df and it is a dataframe

In [14]:
#Problem 6: Drop the 'Value Footnote' column
gdp_df_percapita = gdp_df.drop(columns = ['Value Footnotes'])

In [15]:
gdp_df_percapita

Country or Area  Year        Value
0        Afghanistan  2020  1970.560169
1        Afghanistan  2019  2065.036235
2        Afghanistan  2018  2033.804389
3        Afghanistan  2017  2058.400221
4        Afghanistan  2016  2057.067978
...              ...   ...          ...
7171        Zimbabwe  1994  2603.927369
7172        Zimbabwe  1993  2420.092791
7173        Zimbabwe  1992  2437.152908
7174        Zimbabwe  1991  2733.696626
7175        Zimbabwe  1990  2652.129209

[7176 rows x 3 columns]

In [16]:
#Problem 6 cont: Rename the remaining three columns to 'Country', 'Year', 'GDP_Per_Capita'
gdp_df_percapita.columns = ['Country', 'Year', 'GDP_Per_Capita']

In [17]:
gdp_df_percapita

Country  Year  GDP_Per_Capita
0     Afghanistan  2020     1970.560169
1     Afghanistan  2019     2065.036235
2     Afghanistan  2018     2033.804389
3     Afghanistan  2017     2058.400221
4     Afghanistan  2016     2057.067978
...           ...   ...             ...
7171     Zimbabwe  1994     2603.927369
7172     Zimbabwe  1993     2420.092791
7173     Zimbabwe  1992     2437.152908
7174     Zimbabwe  1991     2733.696626
7175     Zimbabwe  1990     2652.129209

[7176 rows x 3 columns]